### ISSUE TITLE
No way to organize quizzes based on pending reviews

### ISSUE THREAD

**Issue by nrherron92** [gocodebox/lifterlms#2283]
**Is your feature request related to a problem? Please describe.**
HS-202600 

For larger sites that have a lot of quizzes, there is no way to organize the reporting for Quizzes to have those that need grading at the top, leading to having to search through the pagination for any quiz that may need grading. 

**Describe the solution you'd like**
An option on the table to sort by pending review

**Describe alternatives you've considered**
None

**Comment by thomasplevy** [comment]
@nrherron92 good call. I think there's probably a filter type drop-down that could be implemented to filter quizzes by "Status" which would be "All" and "Awaiting Review"

Making the other columns sortable (awaiting review, attempts, and average grade) might not be easy to implement without making a really huge and gross db query. But maybe it is possible by thinking about the data differently and maybe caching awaiting attempt data on the postmeta table or something...

Anyway... This is a good idea that we'll add to the backlog to build out in a future release.
